.github/workflows/readme.yml

```diff
@@ -33,19 +33,14 @@ jobs:
         with:
           files: parsely/build/reports/kover/reportDebug.xml
           token: ${{ secrets.CODECOV_TOKEN }}
-      - name: Validate Maven publication
-        run: ./gradlew :parsely:publishReleasePublicationToMavenLocal
+      - name: Publish snapshot
+        run: ./gradlew :parsely:publishToMavenCentral --max-workers 1
         env:
-          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-          OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-          SIGNING_KEY_ID: ${{ secrets.SIGNING_KEY_ID }}
-          SIGNING_PASSWORD: ${{ secrets.SIGNING_PASSWORD }}
-          SIGNING_KEY: ${{ secrets.SIGNING_KEY }}
-      - name: Publish build artifacts
```

Comment:
Question (❓): Is that really not needed anymore, similar to us publishing build artifacts on `functional-tests`. PS: Which makes me wonder again, why was it needed before. 🤔